spec/support/factory_girl.rb

```diff
@@ -21,4 +21,11 @@
     longitude -0.012345
   end
 
+  factory :meetup do
+    description "Gon' hoop"
+    start_time Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.